src/components/AnimatedLogo.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactComponent as LogoBlades } from "../assets/images/logo-blades.svg";
+import { ReactComponent as LogoBox } from "../assets/images/logo-box.svg";
+
+export default function AnimatedLogo() {
+  return (
+    <div className="logoContainer">
```

Comment:
To match with how the rest of the styles are created (to prevent any unintended css inheriting) could you rename logo Container to 'animatedLogo'?